lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -76,6 +77,10 @@ const isEqualValueSyntaxes = (value1, value2) => {
 		return true;
 	}
 
+	if (!(isStandardSyntaxValue(value1) && isStandardSyntaxValue(value2))) {
+		return false;
+	}
+
 	const value1Node = parse(value1, { context: 'value' });
 	const value2Node = parse(value2, { context: 'value' });
```

Comment:
[suggest] I rethink it safer to catch a syntax error from `css-tree` than our `isStandardSyntaxValue` util. Can you rewrite it like this, please?

```suggestion
	let value1Node;
	let value2Node;

	try {
		value1Node = parse(value1, { context: 'value' });
		value2Node = parse(value2, { context: 'value' });
	} catch (e) {
		return false;
	}
```

Other rules also do the same for `css-tree`, for example:

https://github.com/stylelint/stylelint/blob/369557b79805ca3daee5d595662ba603f27ba9c2/lib/rules/declaration-property-value-no-unknown/index.js#L93-L97

Reply:
Do you want me to move `containsUnsupportedFunction` to utils?

Reply:
No, just catch an error and return `false` from `css-tree`.

Reply:
Oh, sorry. I misunderstood. 😓 

Yes, please move it to utils! 🙏🏼

Reply:
Um, `containsUnsupportedFunction` is needed in this case?

Reply:
In my personal opinion, no, but any suggestion is ok.

Reply:
Very sorry for my wrong request. 🙇🏼 
Could you revert the `containsUnsupportedFunction` change to minimize this PR diff?

Reply:
OK, I’ll do that at tomorrow.